backend-forms.md

```diff
@@ -1081,7 +1081,7 @@ Method | Description
 **addFields** | adds new fields to the outside area
 **addTabFields** | adds new fields to the tabbed area
 **addSecondaryTabFields** | adds new fields to the secondary tabbed area
-**removeField** | remove a field from the tabbed area
+**removeField** | remove a field from the any areas
```

Comment:
Should be just "from any area" not "from the any area"

Reply:
I corrected it.